Jenkinsfile.daily

```diff
@@ -0,0 +1,65 @@
+properties([
+    pipelineTriggers([cron('H H(1-3) * * *')]),
+    buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '30')),
+    disableConcurrentBuilds()
+])
+
+env.gitmirror_node_label = "gitmirror"
+
+stage ("Maintain Reference Repositories") {
+    def nodes = nodesByLabel label:env.gitmirror_node_label
+    nodes.add("master")
+    println "Update Git Mirrors on all nodes with tag ${env.gitmirror_node_label} + master\n${nodes}"
+    def repos = [
+        [
+            url:"https://github.com/ElektraInitiative/libelektra.git",
+            dest: "libelektra"
+        ]
+    ]
+    parallel maintain_repositories("git_mirrors", nodes, repos)
+}
+
+
+def maintain_repositories(basedir, nodes, repos){
```

Comment:
space before `{` or maybe even `(` (like we use it for C). Please have a quick look at doc/CODING.md and ideally define something for Groovy.